deb/Makefile

```diff
@@ -157,21 +157,7 @@ sources/dockerd.json: ../common/dockerd.json
 	    -e 's/$${IMAGE_TAG}/$(IMAGE_TAG)/' \
 	    $< > $@
 
-# TODO: Eventually clean this up when we release an image with a manifest
-DOCKER2OCI=artifacts/docker2oci
-$(DOCKER2OCI):
-	-$(CHOWN) -R $(shell id -u):$(shell id -g) $(@D)
-	docker run --name docker2oci $(GO_IMAGE) sh -c 'go get github.com/coolljt0725/docker2oci'
-	mkdir -p $(@D)
-	docker cp docker2oci:/go/bin/docker2oci "$@"
-	docker rm -f docker2oci
-	$(CHOWN) -R $(shell id -u):$(shell id -g) $(@D)
-
 # offline bundle
 sources/engine.tar: $(DOCKER2OCI)
-	$(MAKE) -C ../image ENGINE_IMAGE=$(ENGINE_IMAGE) ENGINE_SCOPE=$(ENGINE_SCOPE) image-linux
-	mkdir -p artifacts
-	docker save -o artifacts/docker-engine.tar $$(cat ../image/image-linux)
-	./$(DOCKER2OCI) -i artifacts/docker-engine.tar artifacts/engine-image
-	mkdir -p $(@D)
-	tar c -C artifacts/engine-image . > $@
+	$(MAKE) -C ../image ENGINE_IMAGE=$(ENGINE_IMAGE) ENGINE_SCOPE=$(ENGINE_SCOPE) engine-$(ARCH).tar
+	tar c -C ../engine-$(ARCH) . > $@
```

Comment:
I missed this before, but the `engine-$(ARCH).tar` target tars the file already.